main.js

```diff
@@ -0,0 +1,39 @@
+const { app, BrowserWindow } = require("electron");
+const path = require("path");
+
+function createWindow() {
+  const win = new BrowserWindow({
+    width: 800,
+    height: 600,
```

Comment:
can this be configured to be Fullscreen?
can this be configured internally in the app?

Reply:
Yes it can, just remove width and height and add fullscreen as a boolean.

```suggestion
    fullscreen: true,
```

Reply:
More concerned about having the option to tinker with that in configuration than with actually making it fullscreen atm. Open a ticket for that and we can approve this.